site/zenodo_rdm/config.py

```diff
@@ -157,3 +157,4 @@
 #: Email title template.
 SUPPORT_EMAIL_CONFIRM_TITLE_TEMPLATE = "zenodo_rdm/email_confirm_title.html"
 
+ZENODO_BROKER_ENDPOINT = "https://zenodo-broker.web.cern.ch/api/relationships"
```

Comment:
Do we have a local broker that we can point to? Otherwise, I would leave it empty if it doesn't break things so we don't request on "prod" machines from local installations. We should add this as environment variable in Openshift! Can you check a non-breaking default?